src/components/WwwFrame/TheBasketBar.vue

```diff
@@ -69,6 +69,15 @@ export default {
 	computed: {
 		basketLink() {
 			return this.corporate ? this.addHashToRoute('show-basket') : '/basket';
+		},
+		hideBasketBar() {
+			// hide this banner on managed lending landing + checkout pages
+			const routeExclusions = ['/join-team', '/checkout'];
```

Comment:
We have some route exclusions on `TheBasketBar.vue`'s parent, `WwwPage.vue` that determines if basket bar should be on the page. I think these route exclusions would make more sense to go there.

Reply:
Or we consolidate those route rules into this component instead.